persistence/src/commonMain/kotlin/com/wire/kalium/persistence/backup/DatabaseExporter.kt

```diff
@@ -36,8 +52,56 @@ internal class DatabaseExporterImpl internal constructor(
     #define SQLITE_CHECKPOINT_RESTART  2  /* Like FULL but wait for readers */
     #define SQLITE_CHECKPOINT_TRUNCATE 3  /* Like RESTART but also truncate WAL */
      */
-    override fun beforeBackup() {
-        sqlDriver.executeQuery(null, "PRAGMA wal_checkpoint(3)", {}, 0)
+//     override fun beforeBackup() {
+//         sqlDriver.executeQuery(null, "PRAGMA wal_checkpoint(3)", {}, 0)
+//     }
```

Comment:
Remove if we are going to export DB by running migrations on plain DB instead of executing operations on the WAL mode.